packages/web/src/pages/audio-rewards-page/ChallengeRewardsTile.tsx

```diff
@@ -54,20 +58,45 @@ const { fetchUserChallenges, setChallengeRewardsModalType } =
   audioRewardsPageActions
 const { getOptimisticUserChallenges } = challengesSelectors
 
+type ClaimableSummaryTableItem = SummaryTableItem & {
+  claimableDate: dayjs.Dayjs
+  isClose: boolean
+}
+
 const messages = {
   title: 'EARN REWARDS',
   description1: 'Complete tasks to earn $AUDIO tokens!',
   completeLabel: 'COMPLETE',
-  claimReward: 'Claim Your Reward',
+  claimReward: 'Claim This Reward',
   claimAllRewards: 'Claim All Rewards',
-
+  moreInfo: 'More Info',
   readyToClaim: 'Ready to Claim',
+  pendingRewards: 'Pending Reward',
+  totalUpcomingRewards: 'Total Upcoming Rewards',
   totalReadyToClaim: 'Total Ready To Claim',
   pending: 'Pending',
   viewDetails: 'View Details',
   new: 'New!',
   goldAudioToken: 'Gold $AUDIO token',
-  availableNow: '$AUDIO available now'
+  available: '$AUDIO available',
+  now: 'now!',
+  availableMessage: (summaryItems: ClaimableSummaryTableItem[]) => {
```

Comment:
just to nit this is a bit odd, so it seems like we could probably do 
```
const filteredSummaryItems = removeNullable(summaryItems)
const summaryItem = filteredSummaryItems.pop()

if (sumarryItem)....
```

Reply:
i like that